### PR TITLE
CIWEM-15: Add extension skeleton

### PR DESCRIPTION
**Pull request by erawat**
## Overview

This PR adds CiviCRM extension skeleton to the repository.

## Before

The repository was empty.

## After

The repository contains CiviCRM extension skeleton 

## Comments

Extension files were generated by using` civix generate:module` command.

### CODE REVIEW

File: financeextras.php [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+}
+
+/**
+ * Implements hook_civicrm_themes().
+ */
+function financeextras_civicrm_themes(&$themes) {
+  _financeextras_civix_civicrm_themes($themes);
+}
+
+// --- Functions below this ship commented out. Uncomment as required. ---
```
Comment (omarabuhussein) [inline financeextras.php:142]: can you remove these commented out lines.

**Review by omarabuhussein — CHANGES_REQUESTED** [review]
add one note, can you also add the upgrader classes.

File: financeextras.php (hunk 1 above)
Comment (erawat) [reply financeextras.php]: @omarabuhussein  done

**Comment by erawat** [comment]
> add one note, can you also add the upgrader classes.

done!

**Review by omarabuhussein — APPROVED** [review]

**Pull request merged**